integration/v3_grpc_test.go

```diff
@@ -1274,3 +1316,23 @@ func testLeaseRemoveLeasedKey(t *testing.T, act func(*clusterV3, int64) error) {
 		t.Fatalf("lease removed but key remains")
 	}
 }
+
+// TODO: remove the refreshing TTL side-effect
+func leaseExist(t *testing.T, clus *clusterV3, leaseID int64) bool {
```

Comment:
this can probably be done with `LeaseCreate` now that it accepts an optional leaseid